tests/acceptance/view-user-profile-test.js

```
@@ -84,6 +184,7 @@ module('Acceptance | view-user-profile', function (hooks) {
 
     await userPage.visit({ username: 'rohitpaulk' });
     // Completed course will be listed first.
+    await this.pauseTest();
```

ESLint

[error] Do not commit `pauseTest()`

(ember/no-pause-test)